src/pages/pressure-those-in-power/index.js

```diff
@@ -0,0 +1,544 @@
+import React from "react";
+import { OutboundLink } from "gatsby-plugin-google-analytics";
+import Layout from "../../components/layout";
+import ChoiceSupport from "../../components/ChoiceSupport";
+import SEO from "../../components/seo";
+import GlobalStyles from "../../styles/Global.module.scss";
+import ChoiceStyles from "../../styles/Choice.module.scss";
+import ButtonStyles from "../../styles/Buttons.module.scss";
+import Styles from "./Styles.module.scss";
+import ShareSocialCta from "../../components/ShareSocialCta";
+import shareConfig from "../../configs/pages/choiceEatConfig";
+import ChoiceGridOnChoice from "../../components/ChoiceGrid/onChoice";
+import BannerImg from "../../components/Images/Graphics/choiceBannerEat.svg";
```

Comment:
@shylands I know you're a busy man but could you knock out a banner image with a gavel and mark it at 40%? 

> The EU is also close to completing a comprehensive legislative framework that will enable it to implement its commitment under the Paris Agreement to cut emissions by at least 40% by 2030.
https://ec.europa.eu/clima/news/eu-conclude-ratification-second-kyoto-protocol-commitment-period-end-year_en

If not could you help me find somebody who can? I have no idea what to do with a SVG.

Reply:
I can update this image, but also wondering if it might be useful to make these banner svg's a component instead.

We could use something like this https://www.npmjs.com/package/react-minimal-pie-chart to overlay the pie chart on top of the world image.  That would make this stuff configurable without rolling a new SVG from scratch each time

Reply:
@richsoni that would be amazing. Think you could make another PR which either extends this one, or we do that first and then update this one? Either way, this one has been sat around for ages and we should get it merged. It's great work.

Reply:
https://github.com/impactmakers/climatechoice/pull/155/files

Reply:
That pr updates the SVG, and the config file.  I can put together a separate PR after that to do the dynamic pie charts

Reply:
@richsoni thank you! I'll close this one down and move the two remaining bits of feedback over there.